src/hw_mainline.js

```diff
@@ -370,4 +370,23 @@ exports.readPlatform = function (platform) {
         platform.dogtag = fs.readFileSync('/etc/dogtag', 'ascii');
     } catch (ex) {}
     return (platform);
-};
\ No newline at end of file
+};
+
+module.exports = {
+    logfile: logfile,
+    readPWMFreqAndValue: readPWMFreqAndValue,
+    readGPIODirection: readGPIODirection,
+    readPinMux: readPinMux,
+    setPinMode: setPinMode,
+    setLEDPinToGPIO: setLEDPinToGPIO,
+    exportGPIOControls: exportGPIOControls,
+    writeGPIOValue: writeGPIOValue,
+    readGPIOValue: readGPIOValue,
+    enableAIN: enableAIN,
+    readAIN: readAIN,
+    writeGPIOEdge: writeGPIOEdge,
+    writePWMFreqAndValue: writePWMFreqAndValue,
+    readEeproms: readEeproms,
+    readPlatform: readPlatform,
+
```

Comment:
Looks extraneous.

Reply:
Sorry i did not notice this , i copied the exports for the hw_X.js as the exports are same for these files, that was why the mistake was repeated, i will be more careful in future.